### PR TITLE
Keep json libraries up-to-date

### PR DESCRIPTION
**Pull request by psafont**
The usage of yojson and js_of_ocaml was generating deprecation warnings.

Also updated Makefile to use dune instead of jbuilder

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-	jbuilder uninstall
+	dune uninstall
 
 clean:
-	jbuilder clean
+	dune clean
 
 test:
-	jbuilder runtest
+	dune runtest --profile=release
```
Comment (mseri) [inline Makefile]: Is this to avoid failing on warnings?

Comment (psafont) [reply Makefile]: No, I checked other packages and used it this way, I can remove it if it has drawbacks

Comment (mseri) [reply Makefile]: Then let's do without, so we have the tests failing if there are new warnings

**Comment by mseri** [comment]
Thanks!

**Pull request merged**